io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,149 @@
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions":["org.freedesktop.Sdk.Extension.llvm15"],
+    "command": "flatpakLauncher.sh",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/lib/*.la",
+        "/lib/*.a",
+        "/mkspecs",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man",
+        "/opt/cuda/bin",
+        "/opt/cuda/compute-sanitizer",
+        "/opt/cuda/extras",
+        "/opt/cuda/gds",
+        "/opt/cuda/gds-12.2",
+        "/opt/cuda/tools",
+        "/opt/cuda/src",
+        "/opt/cuda/share",
+        "/opt/cuda/pkgconfig",
+        "/opt/cuda/nvvm",
+        "/opt/cuda/nvml",
+        "/opt/cuda/nsight-systems-2023.2.3",
+        "/opt/cuda/nsight-compute-2023.2.1",
+        "/opt/cuda/libnvvp",
+        "/opt/cuda/nsightee_plugins",
+        "/opt/cuda/libnvvp",
+        "/opt/cuda/targets/x86_64-linux/include",
+        "/opt/cuda/targets/x86_64-linux/res",
+        "/opt/cuda/targets/x86_64-linux/lib/*.a",
+        "/opt/cuda/targets/x86_64-linux/lib/libcublas*",
+        "/opt/cuda/targets/x86_64-linux/lib/libcusparse*",
+        "/opt/cuda/targets/x86_64-linux/lib/libcusolver*",
+        "/opt/cuda/targets/x86_64-linux/lib/libcublas*",
+        "/opt/cuda/targets/x86_64-linux/lib/libcurand*",
+        "/opt/cuda/targets/x86_64-linux/lib/libnppif*",
+        "/opt/cuda/targets/x86_64-linux/lib/libnvrtc*",
+        "/opt/cuda/targets/x86_64-linux/lib/stubs",
+        "/opt/cuda/targets/x86_64-linux/lib/cmake",
+        "/opt/intel"
+    ],
+    "modules": [
+        {
+            "name": "oneAPI",
+            "buildsystem": "simple",
+            "build-options":{
+                "append-path": "/usr/lib/sdk/llvm15/bin",
+                "prepend-ld-library-path": "/usr/lib/sdk/llvm15/lib"
+                },
+            "build-commands": [
+                    "chmod +x l_BaseKit_p_2023.2.0.49397_offline.sh",
+                    "./l_BaseKit_p_2023.2.0.49397_offline.sh -x -f webimage_extracted",
+                    "rm l_BaseKit_p_2023.2.0.49397_offline.sh",
+                    "webimage_extracted/l_BaseKit_p_2023.2.0.49397_offline/bootstrapper -s --action=install --components=intel.oneapi.lin.mkl.devel --install-dir=/app/opt/intel/oneapi --eula=accept"
+                    ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397_offline.sh",
+                    "sha256": "3dd04d2cc9ba2a8785580ba09068dbaf9aff9aa930b5fa7f10688210ea9be4a0"
+        
+                }
+            ],
+            "cleanup":[
+                "/app/opt/intel/oneapi/mkl/latest/lib/ia32",
```

Comment:
this doesn't do what you think it does.

>        cleanup (array of strings)
>           An array of file patterns that should be removed at the end. Patterns starting with / are taken to be full pathnames (without the /app prefix),
>           otherwise they just match the basename.

tl;dr the `/` of from the `$FLATPAK_DEST`.

Reply:
thank you; I removed /app from the paths in commit 083be8aa04c0f8951a5521a53eab4f4c0024ac5b